frontend/packages/knative-plugin/src/components/overview/__tests__/KnativeOverview.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import { PodRing } from '@console/shared';
+import { ResourceSummary } from '@console/internal/components/utils';
+import { KnativeOverview } from '../KnativeOverview';
+import { mockOverviewItemData } from './overview-knative-test-data';
+
+describe('KnativeOverview', () => {
+  it('should render PodRing if obj.kind is RevisionModel.kind', () => {
+    const wrapper = shallow(<KnativeOverview item={mockOverviewItemData.item} />);
+    expect(wrapper.find(PodRing)).toHaveLength(1);
+  });
+  it('should render ResourceSummary', () => {
+    const wrapper = shallow(<KnativeOverview item={mockOverviewItemData.item} />);
```

Comment:
It'll goo to have spec do some branch test i.e if item is not of kind revision

Reply:
Done